### ISSUE TITLE
KLL spec 0.3d on Overleaf does not compile/preview

### ISSUE THREAD

**Issue by incredibleweirdo** [kiibohd/kll-spec#2]
Just what it says - visiting the url (https://www.overleaf.com/read/zzqbdwqjfwwf), the latex source is visible on the left pane, but the right pane reports a compile error. 

```
There is no preview for the latest version of this project. The authors of the project have to fix any compile errors before the preview will be available here.
```

Same for the 0.4 link. However, 0.5c seems to compile fine.

**Comment by haata** [comment]
Done. For whatever reason Overleaf makes me log in every couple months to refresh the pdf... :/

Let me know if it's still broken.

**Comment by haata** [comment]
Bah, spoke too soon. Gimme a few.

**Comment by haata** [comment]
Now it's fixed. Turns out I fixed this issue in the KLL 0.5 spec already.

Thanks!
